app/views/users/shows.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Information about a given user</h1>
```

Comment:
- I get an error when I route to this page, this is because you have named this page wrongly. The right name should be `show.html.erb`. Please fix this 🙏